components/renku_data_services/migrations/versions/08ac2714e8e2_add_template_flag_to_projects.py

```diff
@@ -18,7 +18,9 @@
 
 def upgrade() -> None:
     # ### commands auto generated by Alembic - please adjust! ###
-    op.add_column("projects", sa.Column("is_template", sa.Boolean(), default=False, nullable=False), schema="projects")
+    op.add_column(
+        "projects", sa.Column("is_template", sa.Boolean(), server_default=sa.false(), nullable=False), schema="projects"
+    )
```

Comment:
nitpick: this is what `sqlalchemy` generates with the current ORM class.

```suggestion
    op.add_column(
        "projects",
        sa.Column("is_template", sa.Boolean(), server_default=sa.text("false"), nullable=False),
        schema="projects",
    )
```

Reply:
They are equivalent. I've used your suggestion since it's consistent and is used in other places.